data/tokens.json

```diff
@@ -48,7 +48,7 @@
         "name": "Tether USD",
         "symbol": "USDT"
       },
-      "0xCc80C051057B774cD75067Dc48f8987C4Eb97A5e": {
+      "0xcc80c051057b774cd75067dc48f8987c4eb97a5e": {
```

Comment:
why not alway do `toLowerCase` when reading that ?

Reply:
believe me I tried to do something like that, but those are property _names_, not values.  So requires that whole data structure change.